avaframe/data/avaSimilaritySol/Inputs/simiSol_com1DFACfg.ini

```diff
@@ -139,6 +139,8 @@ varParList = massPerPart|dt
 ascendingOrder = False
```

Comment:
general comment ini file: clean those parameters that are not different compared to default